microsoft-edge/webview2/get-started/hololens2.md

```diff
@@ -0,0 +1,594 @@
+
+1. Run the downloaded Mixed Reality Feature Tool executable and follow the prompts to install the MRTK and any additional required packages for HoloLens 2 development.
+
+   The Mixed Reality Feature Tool allows you to select and install the appropriate versions of MRTK, along with other Mixed Reality features, directly into your Unity project.
+
+Proceed to the next step to set up your Unity project for HoloLens 2 development.
+
+
+<!-- ====================================================================== -->
+## Step 3 - Set up Unity project for HoloLens 2 development
+
+Before you can start developing a HoloLens 2 app with WebView2, you'll need to set up your Unity project for HoloLens 2 development. To prepare your project for mixed reality development, follow the steps in [Unity development for HoloLens 2](/windows/mixed-reality/develop/unity/unity-development-overview?tabs=arr,D365,hl2) in the Mixed Reality documentation.
+
+Once you have your Unity project ready for HoloLens 2 development, you'll need to configure MRTK in your project.  To set up the Mixed Reality Toolkit in your Unity project, see [Welcome to the Mixed Reality Feature Tool](/windows/mixed-reality/develop/unity/welcome-to-mr-feature-tool).
+
+After you've installed the MRTK and Mixed Reality Feature Tool, the MRTK assets and prefabs appear in your Unity project, ready for use.
+
+
+<!-- ====================================================================== -->
+## Step 4 - Install Unity WebView plugin via Mixed Reality Feature Tool
+
+In this step, you'll use the Mixed Reality Feature Tool to install the "Microsoft Mixed Reality WebView" plugin that contains a specific version of the WebView2 NuGet package, which is compatible with HoloLens 2.
+
+1. Launch the Mixed Reality Feature Tool (which you installed above). The start page of the Mixed Reality Feature Tool opens, containing options to configure settings, and view documentation:
+
+   ![Mixed Reality Feature Tool start page](./hololens2-images/screenshot_mrtk_feature_tool_start_page.png)
+
+1. Click the gear button.  The **Settings** pane opens.
+
+1. Click the **Feature** tab, and then make sure the **Show preview releases** checkbox is selected:
+
+   ![Mixed Reality Feature Tool settings](./hololens2-images/screenshot_feature_enable_preview_releases.png)
+
+1. Click the **Ok** button to return to the starting page.
+
+1. Click the **Start** button to begin discovering feature packages.
+
+1. Point the Mixed Reality Feature Tool to your Unity project.  To do this, click the **Edit**<!--todo: change "Edit" to string from tooltip--> (**...**) button to the right of the project path field:
+
+   ![Select the Unity project](./hololens2-images/screenshot_feature_tool_select_unity_project.png)
```

Comment:
todo: me or owner -- polish alt-text

alt-text specialized mindset POV:
In a phone call, I'm telling you which png I'm talking about, to differentiate the pngs: 
I'm talking about the png that shows X.

Then set the alt-text to X.

Reply:
i did a pass on alt-text
you can close this thread